axlearn/open_api/metrics/tool_use_execution.py

```diff
@@ -424,6 +424,27 @@ def metric_fn(
 
     Returns:
         A dictionary of metrics.
+        - number_of_examples: Total number of examples.
+        - number_of_parsing_errors: Total number of examples with parsing errors.
+        - number_of_generation_errors: Total number of examples with generation errors.
+        - accuracy: Accuracy after standardization of argument values including removing spaces,
+            punctuation, and converting to lowercases. Also apply matching rules if provided.
+        - strict_accuracy: Strict accuracy where argument values exactly matches the
+            ground truth or not.
+        - lenient_accuracy: Accuracy after removing spaces, punctuation and a static list of
+            stop words in the argument values.
```

Comment:
Point to normalization functions for each type of accuracy?

Reply:
Added.